bookkeeper-server/src/test/java/org/apache/bookkeeper/test/ZooKeeperCluster.java

```diff
@@ -0,0 +1,79 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.bookkeeper.test;
+
+import static org.apache.bookkeeper.util.BookKeeperConstants.AVAILABLE_NODE;
+import static org.apache.bookkeeper.util.BookKeeperConstants.READONLY;
+
+import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.bookkeeper.zookeeper.ZooKeeperWatcherBase;
+import org.apache.zookeeper.CreateMode;
+import org.apache.zookeeper.KeeperException;
+import org.apache.zookeeper.Transaction;
+import org.apache.zookeeper.ZooDefs.Ids;
+import org.apache.zookeeper.ZooKeeper;
+
+/**
+ * Interface for ZooKeeperCluster.
+ */
+public interface ZooKeeperCluster {
+    ZooKeeper getZooKeeperClient();
+
+    String getZooKeeperConnectString();
+
+    String getMetadataServiceUri();
+
+    String getMetadataServiceUri(String zkLedgersRootPath);
+
+    String getMetadataServiceUri(String zkLedgersRootPath, String type);
+
+    void startServer() throws Exception;
```

Comment:
What about renaming to startServers() or startCluster() ?

Reply:
I would suggest `startCluster`. and also change `stopServer`, `restartServer`, `killServer` to `stopCluster`, `restartCluster` and `killCluster`.

Reply:
yeah, but the issue is I've to change in 100's of places where these methods are called in existing testcode..will check

Reply:
I hope your IDE will help....

Reply:
done.